Cargo.toml

```
@@ -8,12 +8,12 @@ edition = "2018"
 
 [dependencies]
 #hashconsing = "1.3"
-hashconsing = { git = "https://github.com/alex-ozdemir/hashconsing.git", branch = "phash"}
+hashconsing = { git = "https://github.com/kwantam/hashconsing.git", branch = "phash"}
```

Ah. Do we still need this change?

---

Ah, right. Let's fix that.

```suggestion
hashconsing = { git = "https://github.com/alex-ozdemir/hashconsing.git", branch = "phash"}
```